src/main/java/controller/UserController.java

```diff
@@ -0,0 +1,62 @@
+package controller;
+import factory.ResponseFactory;
+import factory.UserFactory;
+import model.Request;
+import model.Response;
+import model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import service.UserService;
+import util.ResourceUtils;
+
+public class UserController {
+
+    private static final Logger logger = LoggerFactory.getLogger(UserController.class);
+    private UserService userService;
+
+    public UserController() {
+        userService = new UserService();
+    }
+
+    private Response postUser(Request request) {
+        User user = UserFactory.createUserByBodyParam(request);
+        userService.create(user);
+        return ResponseFactory.createResponse(Integer.parseInt("302"));
```

Comment:
String 에서 int 로 타입변환을 해야하는 특별한 이유가 있나요? 
아래처럼 이 단계를 건너뛰면 안되나요? 
```suggestion
        return ResponseFactory.createResponse(302);
```

아니면 `ResponseFactory.createResponse(int)` 를 `ResponseFactory.createResponse(String)` 으로 수정해도 될거같고요. 빈번한 타입변환은 성능에 영향을 줄 수 있기 때문에 줄일 수 있으면 줄여보는게 좋습니다

Reply:
[9d95ca4](https://github.com/kakao-2022-summer/be-w234-was/pull/59/commits/9d95ca47506a778506117bbb9b2dd4f835fb1bd6) 이것으로 수정